testdata/runtest.sh

```
@@ -1,6 +1,10 @@
 DATABASES="$PREST_PG_DATABASE secondary-db"
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)